ui_testing/testcases/basic_tests/test06_orders.py

```diff
@@ -2296,3 +2297,28 @@ def test065_download_suborder_sheet_for_single_order(self):
                                   f"{str(fixed_sheet_row_data)} : {str(formatted_orders[index])}")
             for item in formatted_orders[index]:
                 self.assertIn(item, fixed_sheet_row_data)
+
```

Comment:
this PR is related to test case 024, so could you remove this test case from her.
`test066_check_order_of_anaylsis_section`